2] loto/main.py

```diff
@@ -0,0 +1,59 @@
+import random
+
+
+def get_computer_numbers():
+    """Get 6 random numbers from the range of 1 – 49
+
+    Returns:
+        list: 6 random numbers from the range of 1 – 49
+    """
+    numbers = []
+    while len(numbers) != 6:
+        n = random.randint(40, 49)
```

Comment:
proc mas range 40 az 49?

Reply:
při testování jsem si zvolil menší range, ať je větší šance na výhru a už jsem to ponechal